README.md

```diff
@@ -44,12 +44,14 @@
 | 8.  | Linear Search | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Searching/linear_search.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Searching/linear_search.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Searching/linear_search.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Searching/linear_search.py) | Searching |
 | 9.  | Bubble Sort | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Sorting/bubble_sort.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/bubble_sort.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Sorting/bubble_sort.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Sorting/bubble_sort.py) | Sorting |
-| 10.  | Heap Sort | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Sorting/heap_sort.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/heap_sort.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Sorting/heap_sort.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Sorting/heap_sort.py) | Sorting |
-| 11.  | Insertion Sort | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Sorting/insertion_sort.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/insertion_sort.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Sorting/insertion_sort.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Sorting/insertion_sort.py) | Sorting |
-| 12.  | Merge Sort | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Sorting/merge_sort.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/merge_sort.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Sorting/merge_sort.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Sorting/merge_sort.py) | Sorting |
-| 13.  | Selection Sort | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Sorting/selection_sort.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/selection_sort.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Sorting/selection_sort.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Sorting/selection_sort.py) | Sorting |
-| 14.  | AVL Tree | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/AVLTree.cpp), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Trees/AVLTree.py) | Trees |
-| 15.  | Decision Tree | [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Machine%20Learning/Decision%20Tree/decision_tree.py) | Machine Learning |
+| 10.  | Cycle Sort |  [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/cycle_sort.cpp) | Sorting |
```

Comment:
<img width="1053" alt="Screenshot 2023-01-02 at 8 16 10 AM" src="https://user-images.githubusercontent.com/76027425/210191985-b0f5dfab-0ad4-474e-8512-1a12f12dfd8c.png">

It is leading to some error in table, please rectify it

Reply:
fixed